pcsx2/MTGS.cpp

```diff
@@ -296,9 +298,23 @@ void SysMtgsThread::MainLoop()
 		// is very optimized (only 1 instruction test in most cases), so no point in trying
 		// to avoid it.
 
+#ifdef PCSX2_CORE
+		if (m_run_idle_flag.load(std::memory_order_acquire) && VMManager::GetState() != VMState::Running)
```

Comment:
Nothing seems to store this flag with `memory_order_release`.  Is that a future or past plan?

Reply:
Whoops, nice catch! SetRunIdle() should indeed be release here.